editor/src/clj/editor/workspace.clj

```diff
@@ -227,6 +214,25 @@ ordinary paths."
       (when-let [workspace (:workspace base-resource)]
         (resolve-workspace-resource workspace path)))))
 
+(defn- template-path [resource-type]
+  (or (:template resource-type)
+      (some->> resource-type :ext (str "templates/template."))))
+
+(defn get-template [workspace resource-type]
```

Comment:
Use `defn-` to make private. Also maybe it should be named `get-template-resource` so it doesn't get confused with the function named `template` below?